modules/core/src/scenegraph/gltf-instantiator/gltf-instantiator.js

```diff
@@ -17,7 +17,8 @@ const ATTRIBUTE_TYPE_TO_COMPONENTS = {
 };
 
 const DEFAULT_OPTIONS = {
-  modelOptions: {}
+  modelOptions: {},
+  pbrDebug: false
```

Comment:
Add to documentation

Reply:
Cannot find the proper .md file. Will do later.